libs/gi/page-team/src/CharacterDisplay/Tabs/TabOptimize/index.tsx

```diff
@@ -156,15 +156,15 @@ export default function TabBuild() {
 
   const [artsDirty, setArtsDirty] = useForceUpdate()
 
-  const [{ threads = defThreads }, setDisplayOptimize] = useState(
+  const [{ threads }, setDisplayOptimize] = useState(
     database.displayOptimize.get()
   )
   useEffect(
     () => database.displayOptimize.follow((_r, to) => setDisplayOptimize(to)),
     [database, setDisplayOptimize]
   )
-
-  const maxWorkers = threads > defThreads ? defThreads : threads
+  const nativeThreads = navigator?.hardwareConcurrency || 4
```

Comment:
@nguyentvan7 
> Can you add a navigator?.hardwareConcurrency || defThreads to wherever this is used in the regular frontend UI?

I did, Its right here.

Reply:
That is just for the dropdown maximum, I was asking for the default selection, here: https://github.com/frzyc/genshin-optimizer/blob/d59b55cbfb785fb2986c0094499c9ea19a8503ad/libs/gi/db/src/Database/DataEntries/DisplayOptimizeEntry.ts#L11

which I guess is also a non-ui library. I still think the max threads should be default, but maybe there's no way to do that

Reply:
I sort of see what you are getting at. I think you have the right idea, we should remove this option from database.